src/it/scala/io/iohk/ethereum/mpt/MerklePatriciaTreeIntegrationSuite.scala

```diff
@@ -1,11 +1,13 @@
 package io.iohk.ethereum.mpt
```

Comment:
These formatting changes are the result of functional changes in the test code and they are required. He just feels sorry it makes a bit harder to review.